src/chrome/content/rules/agoradesk.com.xml

```diff
@@ -0,0 +1,5 @@
+<ruleset name="agoradesk.com">
+	<target host="agoradesk.com" />
+
+	<rule from="^http:" to="https:" />
+</ruleset>
```

Comment:
```suggestion
<ruleset name="agoradesk.com">
	<target host="agoradesk.com" />
	<target host="www.agoradesk.com" />
	<target host="mta-sts.agoradesk.com" />

	<rule from="^http:" to="https:" />
</ruleset>
```

Reply:
mta-sts.agoradesk.com is not needed

Reply:
@LocalMonero Is there some reason for not redirecting it?

Reply:
Not that there's a reason not to redirect it, it's just that it's not for public use. It won't hurt, it's just not _needed_.

Reply:
@LocalMonero Should it be redirected, or removed and marked as "Excluded due to request from website owner"?

Reply:
@pipboy96 just remove that line.

Reply:
@LocalMonero Done, but removing this domain from the ruleset is not enough to hide the existence of this domain, since it can be easily found using domain enumeration tools.

Reply:
@pipboy96 Thanks! We're aware, we were more worried about keeping HTTPS Everywhere's code clean 😄

Reply:
@LocalMonero Are these changes OK? If yes, please press "Commit suggestion", and after that I will request review from another maintainer (I can't merge PRs by myself).